lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -746,21 +773,42 @@ def rotate_axes(xs, ys, zs, zdir):
         return ys, zs, xs
     elif zdir == '-x':
         return zs, xs, ys
-
     elif zdir == 'y':
         return zs, xs, ys
     elif zdir == '-y':
         return ys, zs, xs
-
     else:
         return xs, ys, zs
 
 
+def _rotate_axes_vec(xyz, zdir):
```

Comment:
This will be slower than the original `rotate_axes`, not faster, as it involves copying every element. I'm not sure that it makes sense to vectorize over xyz after all.